cmd/builder/internal/command.go

```diff
@@ -38,7 +38,9 @@ var (
 // Command is the main entrypoint for this application
 func Command() (*cobra.Command, error) {
 	cmd := &cobra.Command{
-		Use: "ocb",
+		SilenceUsage:  true, // Don't print usage on Run error.
+		SilenceErrors: true, // Don't print errors; main does it.
```

Comment:
not relevant to the PR: I remember that I've seen an issue with errors in the collector being printed twice, I feel that this was the issue.